upgrade/upgrade-2.0.4.php

```diff
@@ -0,0 +1,33 @@
+
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+/**
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_2_0_4($module)
```

Comment:
```suggestion
function upgrade_module_2_0_2($module)
```

The latest release is 2.0.1 :sweat_smile:

Reply:
The version into main file is the 2.0.3 😕

Reply:
yes but it's false, https://github.com/PrestaShop/statsforecast/releases

Reply:
Oh wait, I'll check something before

Reply:
Ok, It's 2.0.3 :scream: 
https://packagist.org/packages/prestashop/statsforecast
They did a mess with this repository